rotkehlchen/chain/evm/constants.py

```diff
@@ -5,4 +5,22 @@
 MAX_BLOCKTIME_CACHE = 250  # 55 mins with 13 secs avg block time
 ZERO_ADDRESS = string_to_evm_address('0x0000000000000000000000000000000000000000')
 ETH_SPECIAL_ADDRESS = string_to_evm_address('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE')
-GENESIS_HASH = deserialize_evm_tx_hash('0x' + '0' * 64)  # hash for transactions in genesis block
+EMPTY_64_HEX = '0x' + '0' * 64
```

Comment:
this is an empty 32 byte hash. not 64.

```suggestion
ZERO_32_BYTES_HEX = '0x' + '0' * 64
```